app-network/ucx/autobuild/defines

```diff
@@ -0,0 +1,24 @@
+PKGNAME=ucx
+PKGSEC=science
+PKGDEP="gcc glibc libtool pkg-config rdma-core fuse-3"
+BUILDDEP="autoconf automake m4"
```

Comment:
No need to specify these as they are part of BuildKit.

Reply:
I read this [article](https://wiki.aosc.io/developer/packaging/package-styling-manual/)

> Additional Notes
> When a package has a sole dependency on the GCC Runtime (gcc-runtime) or the GNU C Library (glibc), these dependencies should be included in $PKGDEP.

Maybe I misunderstood it.